src/app/core/services/si/si-core/intacct-connector.service.ts

```
@@ -36,12 +44,71 @@ export class IntacctConnectorService {
+      }),
+      catchError(() => {
+        this.intacctCredential = null;
+        return of({
+          intacctSetupForm: IntacctConnectorModel.mapAPIResponseToFormGroup(null)
+        });
+      })
+    );
+  }
+
+  @Cacheable()
+  checkIntacctTokenHealth(workspaceId: string): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)